testing/BUILD.bazel

```diff
@@ -0,0 +1,22 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+load("@rules_jvm_external//:defs.bzl", "artifact")
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+
+package(default_visibility = ["//visibility:public"])
+
+kt_android_library(
+     name = "testing_main_lib",
+     custom_package = "org.oppia.testing",
+     srcs = glob(["src/main/java/**/FakeSystemClock.kt"]),
```

Comment:
What about the other testing dependencies?

Reply:
Not sure how this code ended up in this PR -- It shouldn't have been included!